tests/large-atoms/input.dat

```diff
@@ -1,124 +1,56 @@
-#! Sample with post-Argon atoms !nosample
+#! Test SCF and DF-MP2 on post-Ar atoms
 
 molecule {
 Ba
-Ba 1 2.0
-units bohr
+Ba 1 4.0
 }
 
 set {
-  basis def2-sv_p_
-  scf_type df
-  DF_BASIS_SCF def2-sv_p_-jkfit
-}
-
-analytic_grad = gradient('mp2')
-clean()
-
-
-molecule {
-Ba
-Ba 1 2.0
-units bohr
-}
-
-set {
-  basis def2-sv_p_
-  scf_type df
-  DF_BASIS_SCF def2-sv_p_-jkfit
-}
-
-analytic_grad = gradient('scf')
-clean()
-
-
-molecule {
-In
-In 1 2.0
-units bohr
+  basis def2-svp
+  points 5
 }
 
-set {
-  basis STO-3G
-  scf_type pk
-}
+ref_energy =  -50.11011922912507 #TEST
 
-thisenergy = energy('mp2')
+analytic_grad, wfn = gradient('scf', return_wfn=True)
+this_energy = wfn.energy()
+findif_grad = gradient('scf', dertype=0)
+assert compare_values(findif_grad, analytic_grad, 6, "Analytic vs. FinDiff Gradient (SCF)") #TEST
+assert compare_values(ref_energy, this_energy, 6, "Calculated vs. Reference Energy (SCF)") #TEST
```

Comment:
ping me if the pytest conversion is entirely foreign. roughly, you'll want to parameterize each of the gradient jobs with a different mol, method, options (basis, etc.), and ref value.

mol is trickiest, and you can use https://github.com/psi4/psi4/blob/master/tests/pytests/test_weird_basis.py#L3-L228 as a model, ignoring all the specialty basis set stuff. 

thanks for working on this!